src/providers/completionProvider.ts

```diff
@@ -370,3 +387,21 @@ function atEndOfLine(document: TextDocument, position: Position): boolean {
   ];
   return charAfterCursor === "\n" || charAfterCursor === "\r";
 }
+
+function resolveSuffix(optionType: string) {
+  let returnSuffix: string;
+
+  switch (optionType) {
+    case "list":
+      returnSuffix = "\r\t- ";
+      break;
+    case "dict":
+      returnSuffix = "\r\t\t";
```

Comment:
Same here - I'm ending up with extra indentation.

Reply:
So the idea behind the extra `\t` is as follows:

If a dict is a first element of a list then only `\r\t` would put the cursor at the next line under the same level of the option which is not correct (since, we want to place the cursor one level deep of that option). This is because of the extra `-` that comes from the list.

But if the option is not the first element of the list, it works fine.

So, to overcome this problem, dicts are suffixed with `\r\t\t` (double tabs).

Anyways, I have added logic to identify if an option is the first element of the list or not. And accordingly indentations are provided. You can now take a look.

Reply:
Works like a charm now :)